src/wwwroot/config/config.context/local.json

```diff
@@ -1,2 +1,3 @@
 {
+    "serverUrl" : "http://dws02.edu.isd.dp.ua:9000/"
```

Comment:
these changes have to be only local. please discard this changes from pull request;